sources/Makefile

```diff
@@ -241,24 +241,44 @@ workers:
 # Enable local DB connection to Openshift DEV/TEST/STG/PROD
 connect-dev-db:
 	@echo "Forward local port 5454 to Openshift simsdb-pgbouncer default port."
-	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5454."
+	@echo "Use your preferred DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5454."
 	@oc project 0c27fb-dev
 	@oc port-forward services/simsdb-pgbouncer 5454:5432
 
 connect-test-db:
 	@echo "Forward local port 5555 to Openshift simsdb-pgbouncer default port."
-	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5555."
+	@echo "Use your preferred DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5555."
 	@oc project 0c27fb-test
 	@oc port-forward services/simsdb-pgbouncer 5555:5432
 
 connect-stg-db:
 	@echo "Forward local port 5656 to Openshift simsdb-pgbouncer default port."
-	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5656."
+	@echo "Use your preferred DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5656."
 	@oc project a6ef19-test
 	@oc port-forward services/simsdb-pgbouncer 5656:5432
 
 connect-prod-db:
 	@echo "Forward local port 5757 to Openshift simsdb-pgbouncer default port."
-	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5757."
+	@echo "Use your preferred DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5757."
 	@oc project 0c27fb-prod
 	@oc port-forward services/simsdb-pgbouncer 5757:5432
+connect-dev-db-superuser:
+	@echo "Forward local port 5454 to Openshift master pod default port."
```

Comment:
Please do not reuse the same local ports. Using different ports would allow the local connections to be pre-configured.